client/src/lib/dependency-injection/buildReducerContainer.js

```diff
@@ -11,8 +11,6 @@ const buildReducerContainer = (base = buildBaseContainer()) => ({
   store: null,
 
   setStore(store) {
-    this.isProtected();
```

Comment:
@unclecheese can you double check this logic is ok? It gets my approval if so.

Reply:
It's probably okay, but I'm wondering if there is truly no viable case for a user replacing the store post-boot, we should still protect it.

What if we added a second property (e.g. `frozen`) that got set when the `ready()` callbacks finished, which would allow us to protect methods less strictly?

There is a good use case for replacing the store (i.e. adding new middleware), but we want to enforce that it happens post init, pre boot. So we may need more state.